apps/web/pages/event-types/index.tsx

```diff
@@ -115,30 +117,45 @@ const MobileTeamsTab: FC<MobileTeamsTabProps> = (props) => {
 const Item = ({ type, group, readOnly }: { type: EventType; group: EventTypeGroup; readOnly: boolean }) => {
   const { t } = useLocale();
 
-  return (
+  const content = () => (
+    <div>
+      <span
+        className="font-semibold text-gray-700 ltr:mr-1 rtl:ml-1"
+        data-testid={"event-type-title-" + type.id}>
+        {type.title}
+      </span>
+      {group.profile.slug ? (
+        <small
+          className="hidden font-normal leading-4 text-gray-600 sm:inline"
+          data-testid={"event-type-slug-" + type.id}>
+          {`/${
+            type.schedulingType !== SchedulingType.MANAGED ? group.profile.slug : t("username_placeholder")
+          }/${type.slug}`}
+        </small>
+      ) : null}
+      {readOnly && (
+        <Badge variant="gray" className="ml-2">
+          {t("readonly")}
+        </Badge>
+      )}
+    </div>
+  );
+
+  return readOnly ? (
```

Comment:
Fixes #7882

Reply:
Shouldn't we rather let the user open it to see all event-type details and just not allow editing? Can be done in a follow-up